tests/examples/wibox/awidget/prompt/highlight.lua

```diff
@@ -0,0 +1,66 @@
+local parent    = ... --DOC_NO_USAGE --DOC_HIDE
+local wibox     = require( "wibox"     ) --DOC_HIDE
+local awful     = { prompt = require("awful.prompt") }--DOC_HIDE
+local beautiful = require( "beautiful" ) --DOC_HIDE
+
+    local amp = "&amp"..string.char(0x3B)
+    local quot = "&quot"..string.char(0x3B)
```

Comment:
At first I tried to have them in the doc, but ldoc isn't advanced enough to do that. Also, the little magic syntax to add the boilerplate is necessary, or it would require an external template for each test, making things worst instead of better.

I am not saying it is impossible to improve the current system. It is. What I am saying is that before saying "lets ditch it and start over", the replacement has to be *better* than the current solution.

> Hasn't my tests-cleanup broke those then?!

I hope not, some documentation pages are entirely written using it.

Once that "improve CI accuracy" PR is merged, I will open a new one to fix the other @psychon issue about the lines being off in users backtraces. This will also solve the speed issue as it wont run everytime cmake is called. The next steps would be to generate the test list using cmake, then have a single lua process generate all of them. I have a prototype of that, but it has its own share of issues.

As for having them "in" the lua file. It isn't impossible (and it is probably a good thing), but it will take a lot more work than you seem to think to get to the "status quo".